src/tensor/domain.ts

```diff
@@ -0,0 +1,119 @@
+import { PublicKey } from "@solana/web3.js";
+import { Attribute } from "./types";
+
+// Do these really need to be constants??
+const DIGITS_ONLY = "Digits Only";
+const LETTERS_ONLY = "Letters Only";
+const PALINDROME = "Palindrome";
+const EMOJI = "Emoji";
+
+export const DOMAIN_MCC_LIST = [
+  "E5ZnBpH9DYcxRkumKdS4ayJ3Ftb6o3E8wSbXw4N92GWg",
+  "86deDknZeDhko46gB8SqK7rYc5HnSBjKDvo6Mi7viYS9",
+  "6bsj8ybPa9xsc6pcAme4x6LvhKvtCmgA4TwwG4qtFw5Z",
+  "GYLiNNu4pqL6QvZKYHW2EMoibVFm2aVJsPHpUVLcU6pL",
+  "7yQYe84W7a5VgNvtRzsvy7mPRed5gmL9HnvJfsbPWK9J",
+];
+
+export const languageRegex = {
```

Comment:
better to have it like this and do object.entries etc etc?
Or 2d array and a forEach?